packages/block-editor/src/components/rich-text/index.js

```diff
@@ -39,6 +39,34 @@ import { useFormatTypes } from './use-format-types';
 import FormatEdit from './format-edit';
 import { getMultilineTag, getAllowedFormats } from './utils';
 
+/**
+ * Removes props used for the native version of RichText so that they are not
+ * passed to the DOM element and log warnings.
+ *
+ * @param {Object} props Props to filter.
+ *
+ * @return {Object} Filtered props.
+ */
+function removeNativeProps( props ) {
+	return omit( props, [
+		'__unstableMobileNoFocusOnMount',
+		'deleteEnter',
+		'placeholderTextColor',
+		'textAlign',
+		'selectionColor',
+		'tagsToEliminate',
+		'rootTagsToEliminate',
+		'disableEditingMenu',
+		'fontSize',
+		'fontFamily',
+		'fontWeight',
+		'fontStyle',
+		'minWidth',
+		'maxWidth',
+		'setRef',
```

Comment:
Could we use `ref` here instead?

Reply:
If we use `ref` we would get the reference of the `RichText` component, right? If that's the case, it won't work because here what we need is the reference of the underneath native view of Aztec (`RCTAztecView`).

Reply:
Good point, yeah, we could forward the ref upstream instead of using `setRef` 👍 .